docs/initrd.md

```diff
@@ -0,0 +1,10 @@
+To build an image with a mkosi-built initrd:
+1. Follow the steps described [here](https://github.com/systemd/mkosi-initrd/blob/main/docs/fedora.md)
```

Comment:
Can we inline this in this document and make it non-fedora specific?

I would just replace this with the most minimal config we can find that produces a working initrd.